doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -257,3 +296,21 @@ Adjust frame(s) of selected key(s)              :kbd:`Shift` + click and drag le
 Adjust value(s) of selected key(s)              :kbd:`Shift` + click and drag up or down
 =============================================== =============================================
 ```
+
+
+## Plug Values ##
```

Comment:
I know we show plug values elsewhere too, but I wonder if it's worth calling this section "Node Editor", since all the other top level sections correspond to a specific Editor type.

Reply:
Users have pointed out to me other interface actions common to certain field types, like selecting whole columns/rows in tables. Maybe we should consider a "General" section, containing "Numeric Plugs" and "Text Plugs"? The latter would be a good spot to put the path actions I didn't know about. Or is that too granular?

Reply:
I think NodeEditor would be a good top-level section to enclose subsections for "Numeric Plugs" and "Text Plugs". The other main sections are for editors and also have subsections so it seems to fit. The other thing we could mention in there is the right click context menu, and dragging and shift dragging the label to drag connections and values.

 A separate "General" top-level section might also make sense for documenting general conventions...

Reply:
The other nice thing about top-level sections for each editor is that they'd make very natural places to link to directly from the UI.